### PR TITLE
Add HSTS configuration docs

### PR DESCRIPTION
**Pull request by bangbingsyb**
Drafted the configuration docs for HSTS. Please enable preview so that other team member could review the draft based on the preview as well.

### CODE REVIEW

**Comment by shirhatti** [comment]
Can you not dupe the content and samples?
I'd prefer to leave the <site>/<hsts> content in there. As for the the <siteDefaults>/<hsts>, I'd rather just add a redirect or a dupe link so that it shows up in the TOC in the right place, but I really don't want to two copies of the sample article.

File: iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<site name="Contoso" id="1">
```
Comment (shirhatti) [inline iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml]: Nit: Can you give a more meaningful file name for all of these samples?

Comment (shirhatti) [inline iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml]: Perhaps as simple as `hsts.xml`

Comment (bangbingsyb) [inline iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml]: The two hsts.md files correspond to the HSTS configuration under site and siteDefaults, respectively, and they have different contents and sample config/codes. So they cannot be combined.

For the name of the samples, I followed the naming conventions of the existing ones. For example, check iis-docs\iis\configuration\system.applicationHost\sites\site\limits\samples. Not a fan of this naming convention, either, but the file extensions provide good indicate about each sample:

.xml -  sample config
.cmd - appcmd commands
.cs - C#
...

Comment (bangbingsyb) [reply iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml]: Any further comment?

Comment (bangbingsyb) [reply iis/configuration/system.applicationHost/sites/site/hsts/samples/sample1.xml]: Could you help merge the PR?

**Comment by shirhatti** [comment]
Merged in 1d7a3666a288c8b6a00b50f1d5b49a3f570401dd

**Pull request closed without merging**